examples/manila-provisioner/csi-cephfs/services.yaml

```diff
@@ -23,16 +23,4 @@ spec:
   ports:
     - name: dummy
       port: 12345
----
```

Comment:
The services are there because they're required by `StatefulSet` deployments for external-attacher and provisioner sidecars, but manila-provisioner is deployed as a `Deployment` so no service neede.